itests/kit2/chain.go

```diff
@@ -0,0 +1,21 @@
+package kit2
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/stretchr/testify/require"
+)
+
+func WaitTillChainHeight(ctx context.Context, t *testing.T, node *TestFullNode, blocktime time.Duration, height int) abi.ChainEpoch {
```

Comment:
We should have this as a `TestFullNode` method. Then we can remove the `t` and `node` arguments.

Reply:
Also, let's document the testkit. We are really trying to up the quality bar to make it easy for future test writers to use this framework!